scene/2d/camera_2d.cpp

```diff
@@ -699,6 +703,7 @@ void Camera2D::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("_set_old_smoothing", "follow_smoothing"), &Camera2D::_set_old_smoothing);
 
+#ifdef TOOLS_ENABLED
```

Comment:
That's problematic as it means users can use these methods/properties in their script when in the editor, but it will then error at runtime once exported.

Reply:
Then may just compile out function's contents instead?

Reply:
You could do that (we have some methods like this), but I wonder if it's worth it. The impact on performance/size is minimal and the code will be cluttered with #ifdefs.

Reply:
Changes made. I honestly don't really find it cluttering.

Reply:
Ah, I thought there would be more. Like, you could remove the "internal" properties too.